book/book.tex

```diff
@@ -3806,7 +3807,7 @@ \subsection{FIFO barbershop solution}
 This solution is in \verb"sync_code/barber3.py" (see~\ref{sync.py}).
 
 The only way I can think of to solve this problem is to create a third
-kind of thread, which I can an usher.  The ushers manage {\tt queue1}
+kind of thread, which I can name it an usher.  The ushers manage {\tt queue1}
```

Comment:
Can I ask you to change this to "which I'll call an usher"?

Or I can merge this and then make the change myself.

Reply:
Sounds better. 
Done.